test/unit/BCoWPool/BCoWPool.tree

```diff
@@ -1,3 +1,10 @@
+BCoWPool::Constructor
+└── when called
+    ├── it should set the solution settler
+    ├── it should set the domain separator
+    ├── it should set the vault relayer
```

Comment:
missing the external call? `should query solution settler for vault relayer` + `should query solution settler for app data`

Reply:
"ask" doesn't sound good